draft-irtf-qirg-principles-03.xml

```diff
@@ -795,6 +920,37 @@
            information via separate classical channels which the repeaters will
            have to correlate with the qubits stored in their memory.</t>
 
+           <t> First generations of quantum networks will not be store-and-forward
+           networks.
+           <vspace blankLines="1" />
+
+           As described in <xref target="elg"></xref>,
+           Quantum links provide Bell pairs that are undirected network
+           resources, in contrast to directed links of classical networks.
```

Comment:
```suggestion
           resources, in contrast to directed packets of classical networks.
```

Reply:
links don't have to be directional in classical networks either - it is packets/frames/datagrams that have a source and destination

Reply:
```suggestion
           resources, in contrast to directed frames of classical networks.
```
I think this should be frames because this sentence talk about links.